packages/components/src/components/hds/app-frame/index.ts

```diff
@@ -53,6 +56,11 @@ export default class HdsAppFrame extends Component<HdsAppFrameSignature> {
     return this.args.hasModals ?? true;
   }
 
+  // Indicates if the "sticky footer" container should be displayed
+  get hasStickyFooter(): boolean {
+    return this.args.hasStickyFooter ?? true;
```

Comment:
`false` might be a more sensible default here, given this is a new component.

Reply:
thought: I tend to agree with Phil here, maybe it should be an opt-in. although I see how it will not harm if it's true but doesn't have content